scripts/cleanbox

```diff
@@ -50,7 +50,7 @@ echo "${NET}.2    server.rudder.local server rudder" >> /etc/hosts
 i=2
 for host in ${HOSTS}
 do
-  echo "${NET}.${i}    ${host}" >> /etc/hosts
+  echo "${NET}.${i}    ${host} ${host}.normation.com" >> /etc/hosts
```

Comment:
I don't think rudder-test uses .normation.com as the domain name for machines.
And if it does, it probably shouldn't